src/ast/data_type.rs

```diff
@@ -140,7 +140,10 @@ impl fmt::Display for DataType {
             DataType::Text => write!(f, "TEXT"),
             DataType::String => write!(f, "STRING"),
             DataType::Bytea => write!(f, "BYTEA"),
-            DataType::Array(ty) => write!(f, "{}[]", ty),
+            DataType::Array(ty) => match **ty {
+                DataType::Text => write!(f, "{}[]", ty),
```

Comment:
Why `Text` case is special?

Reply:
This is the syntax of `PostgreSQL`, maybe we don't need to be compatible with it.
https://www.postgresql.org/docs/current/arrays.html

Reply:
Yes, it's fixed size array, we do not support it for now. 

Just `Arary(ty)`